src/rgw/rgw_op.cc

```diff
@@ -3532,6 +3542,9 @@ void RGWPutObj::execute()
     }
   }
 
+#ifdef WITH_LTTNG
+  tracepoint(rgw_op, before_data_transfer, s->req_id.c_str());
+#endif
```

Comment:
it doesn't look like these `#ifdef WITH_LTTNG` blocks are necessary around `tracepoint()` calls - when `WITH_LTTNG` isn't defined, `#define tracepoint(...)` means that the call will be removed by the preprocessor

Reply:
@cbodley You are right. :)